src/Glpi/Http/SessionManager.php

```diff
@@ -79,10 +79,10 @@ public function isResourceStateless(Request $request): bool
         $path = $this->normalizePath($request);
 
         $path_matches = [];
-        $plugin_path_pattern = '#^/plugins/(?<plugin_key>[^/]+)(?<plugin_resource>/.+)$#';
+        $plugin_path_pattern = '#^/plugins/(?<plugin_key>[^/]+)(?<plugin_resource>/.+)?$#';
```

Comment:
In #20652, I assumed that a plugin route cannot be an empty string. Is thre a real case for this? In this case, the `src/Glpi/Kernel/Listener/RequestListener/PluginsRouterListener.php` should probably updated too.

Reply:
This is mandatory for inventory plugin to handle the following URL as a stateless route: `http://glpi/plugins/glpiinventory/`.

PluginsRouterListener has been changed in #20652